### ISSUE TITLE
PyCall on windows throws shlib_ext missing error

### ISSUE THREAD

**Issue by ibell** [stevengj/PyCall.jl#125]
```
C:\Users\Belli\Desktop>julia runme.jl
ERROR: LoadError: LoadError: LoadError: UndefVarError: shlib_ext not defined
 in include at boot.jl:250
 in include_from_node1 at loading.jl:129
 in include at boot.jl:250
 in include_from_node1 at loading.jl:129
 in reload_path at loading.jl:153
 in _require at loading.jl:68
 in require at loading.jl:51
 in include at boot.jl:250
 in include_from_node1 at loading.jl:129
 in process_options at client.jl:318
 in _start at client.jl:402
while loading C:\Users\Belli\.julia\v0.4\PyCall\src\pyinit.jl, in expression starting on line 13
while loading C:\Users\Belli\.julia\v0.4\PyCall\src\PyCall.jl, in expression starting on line 133
while loading C:\Users\Belli\Desktop\runme.jl, in expression starting on line 1
```

**Comment by stevengj** [comment]
This is recent breakage on the 0.4 branch, and should be fixed by #124.

In general, you shouldn't be using Julia 0.4 at this point unless you are a developer, because it is breaking things continually.

**Comment by ibell** [comment]
Yes, I've noticed :)  But other things were broken on 0.3.6 which is why I
have to use 0.4
